.travis.yml

```diff
@@ -7,3 +7,19 @@ ghc:
   - "8.2"
   - "8.0"
   - "7.10"
+
+before_deploy: |
+  if [[ $TRAVIS_HASKELL_VERSION == "8.6.5" ]]; then
+    tar -C dist/build/dotenv -cvf dotenv-linux-x86_64-static.tar.gz dotenv
+    cabal install -f 'static'
```

Comment:
@CristhianMotoche perhaps I'm missing something here but, I think you need to compile the package again with` -f static` on in order to produce a static binary.

Reply:
Hi @sestrella I think i accidentally switched lines 13 and 14. BTW, I'm not sure if we need to build the whole package again with `-f static`. I was testing locally and doing a `cabal install` produces the static binary.